
from PySide2.QtCore import *
from PySide2.QtGui import *
from PySide2.QtWidgets import *


class Ui_DataInput(object):
    def setupUi(self, DataInput):
        if not DataInput.objectName():
            DataInput.setObjectName(u"DataInput")
        DataInput.resize(455, 755)
        self.centralwidget = QWidget(DataInput)
        self.centralwidget.setObjectName(u"centralwidget")
        self.frame = QFrame(self.centralwidget)
        self.frame.setObjectName(u"frame")
        self.frame.setGeometry(QRect(0, 0, 450, 750))
        self.frame.setStyleSheet(u"QFrame {	\n"
"	background-color: rgb(56, 58, 89);	\n"
"	color: rgb(220, 220, 220);\n"
"	border-radius: 10px;\n"
"}")
        self.frame.setFrameShape(QFrame.StyledPanel)
        self.frame.setFrameShadow(QFrame.Raised)
        self.plainTextEditWeight = QPlainTextEdit(self.frame)
        self.plainTextEditWeight.setObjectName(u"plainTextEditWeight")
        self.plainTextEditWeight.setGeometry(QRect(190, 370, 231, 51))
        font = QFont()
        font.setPointSize(16)
        self.plainTextEditWeight.setFont(font)
        self.labelLname = QLabel(self.frame)
        self.labelLname.setObjectName(u"labelLname")
        self.labelLname.setGeometry(QRect(10, 120, 181, 51))
        font1 = QFont()
        font1.setBold(True)
        font1.setWeight(75)
        self.labelLname.setFont(font1)
        self.labelLname.setStyleSheet(u"color: rgb(90, 93, 255);")
        self.plainTextEditLname = QPlainTextEdit(self.frame)
        self.plainTextEditLname.setObjectName(u"plainTextEditLname")
        self.plainTextEditLname.setGeometry(QRect(190, 120, 231, 51))
        self.plainTextEditLname.setFont(font)
        self.labelWeight = QLabel(self.frame)
        self.labelWeight.setObjectName(u"labelWeight")
        self.labelWeight.setGeometry(QRect(10, 370, 181, 51))
        self.labelWeight.setFont(font1)
        self.labelWeight.setStyleSheet(u"color: rgb(90, 93, 255);")
        self.labelHeight = QLabel(self.frame)
        self.labelHeight.setObjectName(u"labelHeight")
        self.labelHeight.setGeometry(QRect(10, 300, 181, 51))
        self.labelHeight.setFont(font1)
        self.labelHeight.setStyleSheet(u"color: rgb(90, 93, 255);")
        self.plainTextEditHeight = QPlainTextEdit(self.frame)
        self.plainTextEditHeight.setObjectName(u"plainTextEditHeight")
        self.plainTextEditHeight.setGeometry(QRect(190, 300, 231, 51))
        self.plainTextEditHeight.setFont(font)
        self.labelGroupName = QLabel(self.frame)
        self.labelGroupName.setObjectName(u"labelGroupName")
        self.labelGroupName.setGeometry(QRect(0, 640, 441, 41))
        self.labelGroupName.setStyleSheet(u"color: rgb(255, 255, 255);")
        self.pushButtonEnter = QPushButton(self.frame)
        self.pushButtonEnter.setObjectName(u"pushButtonEnter")
        self.pushButtonEnter.setGeometry(QRect(110, 540, 200, 50))
        font2 = QFont()
        font2.setFamily(u"Calibri")
        font2.setPointSize(20)
        self.pushButtonEnter.setFont(font2)
        self.plainTextEditAge = QPlainTextEdit(self.frame)
        self.plainTextEditAge.setObjectName(u"plainTextEditAge")
        self.plainTextEditAge.setGeometry(QRect(190, 240, 231, 51))
        self.plainTextEditAge.setFont(font)
        self.labelAge = QLabel(self.frame)
        self.labelAge.setObjectName(u"labelAge")
        self.labelAge.setGeometry(QRect(10, 240, 181, 51))
        self.labelAge.setFont(font1)
        self.labelAge.setStyleSheet(u"color: rgb(90, 93, 255);")
        self.labelFname = QLabel(self.frame)
        self.labelFname.setObjectName(u"labelFname")
        self.labelFname.setGeometry(QRect(10, 60, 181, 51))
        self.labelFname.setFont(font1)
        self.labelFname.setStyleSheet(u"color: rgb(90, 93, 255);")
        self.plainTextEditFname = QPlainTextEdit(self.frame)
        self.plainTextEditFname.setObjectName(u"plainTextEditFname")
        self.plainTextEditFname.setGeometry(QRect(190, 60, 231, 51))
        self.plainTextEditFname.setFont(font)
        self.labelGender = QLabel(self.frame)
        self.labelGender.setObjectName(u"labelGender")
        self.labelGender.setGeometry(QRect(10, 180, 181, 51))
        self.labelGender.setFont(font1)
        self.labelGender.setStyleSheet(u"color: rgb(90, 93, 255);")
        self.plainTextEditGender = QPlainTextEdit(self.frame)
        self.plainTextEditGender.setObjectName(u"plainTextEditGender")
        self.plainTextEditGender.setGeometry(QRect(190, 180, 231, 51))
        self.plainTextEditGender.setFont(font)
        self.labelTime = QLabel(self.frame)
        self.labelTime.setObjectName(u"labelTime")
        self.labelTime.setGeometry(QRect(10, 450, 181, 51))
        self.labelTime.setFont(font1)
        self.labelTime.setStyleSheet(u"color: rgb(90, 93, 255);")
        self.plainTextEditTime = QPlainTextEdit(self.frame)
        self.plainTextEditTime.setObjectName(u"plainTextEditTime")
        self.plainTextEditTime.setGeometry(QRect(190, 450, 231, 51))
        self.plainTextEditTime.setFont(font)
        DataInput.setCentralWidget(self.centralwidget)

        self.retranslateUi(DataInput)

        QMetaObject.connectSlotsByName(DataInput)
    # setupUi

    def retranslateUi(self, DataInput):
        DataInput.setWindowTitle(QCoreApplication.translate("DataInput", u"MainWindow", None))
        self.plainTextEditWeight.setPlaceholderText(QCoreApplication.translate("DataInput", u"in kilograms", None))
        self.labelLname.setText(QCoreApplication.translate("DataInput", u"<html><head/><body><p align=\"center\"><span style=\" font-size:16pt;\">Last Name</span></p></body></html>", None))
        self.plainTextEditLname.setPlaceholderText(QCoreApplication.translate("DataInput", u"enter here", None))
        self.labelWeight.setText(QCoreApplication.translate("DataInput", u"<html><head/><body><p align=\"center\"><span style=\" font-size:16pt;\">Weight</span></p></body></html>", None))
        self.labelHeight.setText(QCoreApplication.translate("DataInput", u"<html><head/><body><p align=\"center\"><span style=\" font-size:16pt;\">Height</span></p></body></html>", None))
        self.plainTextEditHeight.setPlaceholderText(QCoreApplication.translate("DataInput", u"in centimeters", None))
        self.labelGroupName.setText(QCoreApplication.translate("DataInput", u"<html><head/><body><p align=\"center\"><span style=\" font-size:14pt; font-weight:600;\">Group 76_Treadmill Dashboard</span></p></body></html>", None))
        self.pushButtonEnter.setText(QCoreApplication.translate("DataInput", u"Enter", None))
        self.plainTextEditAge.setPlaceholderText(QCoreApplication.translate("DataInput", u"enter here", None))
        self.labelAge.setText(QCoreApplication.translate("DataInput", u"<html><head/><body><p align=\"center\"><span style=\" font-size:16pt;\">Age</span></p></body></html>", None))
        self.labelFname.setText(QCoreApplication.translate("DataInput", u"<html><head/><body><p align=\"center\"><span style=\" font-size:16pt;\">First Name</span></p></body></html>", None))
        self.plainTextEditFname.setPlaceholderText(QCoreApplication.translate("DataInput", u"enter here", None))
        self.labelGender.setText(QCoreApplication.translate("DataInput", u"<html><head/><body><p align=\"center\"><span style=\" font-size:16pt;\">Gender</span></p></body></html>", None))
        self.plainTextEditGender.setPlaceholderText(QCoreApplication.translate("DataInput", u"M or F", None))
        self.labelTime.setText(QCoreApplication.translate("DataInput", u"<html><head/><body><p align=\"center\"><span style=\" font-size:16pt;\">Time</span></p></body></html>", None))
        self.plainTextEditTime.setPlaceholderText(QCoreApplication.translate("DataInput", u"in minutes", None))
    # retranslateUi

